.github/workflows/build-and-test-ee.yml

```
@@ -0,0 +1,24 @@
+name: Build and Test EE
+on: [pull_request]
+jobs:
+  build-and-test-ee:
+    if: github.event.pull_request.head.repo.full_name == github.repository
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        include:
+          - DISTRO: "tomcat"
+            EE: "true"
+          - DISTRO: "wildfly"
+            EE: "true"
+          - DISTRO: "run"
+            EE: "true"
```

There's not really a need for a complex matrix if EE is always `true` for this workflow.

```suggestion
        DISTRO: [tomcat, wildfly, run]
```